src/pages/content/word-list.mdx

```diff
@@ -1,12 +1,707 @@
+
+Hyphenate sign-on and spell this out on first use.
+
+### spam
+
+Don’t use spam as a verb.
+
+### subdomain
+
+One word.
+
+### subscription
+
+For a person’s Zendesk account only. Not for individual plans or products.
+
+### support, Support
+
+Capitalize when referring to the Zendesk Support product. Use lowercase
+otherwise.
+
+### Support team
+
+Don’t use. Use Zendesk Customer Support or Customer Support instead.
+
+### stats
+
+Used in Help Center articles and in reports. Abbreviation OK.
+
+### thank you, thanks
+
+Reserve thank you and thanks when users provide information that’s difficult or
+inconvenient to collect.
+
+<BestPractice>
+  <Do>
+    <p>Thank you for submitting a request.</p>
+  </Do>
+  <Dont>
+    <p>Thank you for changing your settings.</p>
+  </Dont>
+</BestPractice>
+
+### ticket, ticket form
+
+Lowercase for both. Don’t capitalize for emphasis.
+
+### trigger
+
+Lowercase.
+
+### user name
+
+Two words. Not hyphenated.
+
+### website, web form, web page
+
+All lowercase.
+
+### Web Widget
+
+Capitalize both words.
+
+### workflow
+
+One word.
+
+### Zendesk
+
+Always capitalize the name of our company. Never use ZenDesk or give it an
+apostrophe. We’re not the possessive type.
+
+### Zendesk Apps
+
+Capitalize this feature name: “The Zendesk Apps framework.”
+
+<hr />
+
+## Words we avoid
+
+These words aren’t bad. They’re just not in our terminology diet.
+
+### oops, whoops, oh no
+
+Oops, we did it again. Seriously, though, we don’t use oops or other
+self-deprecating blurts in error messages or empty states.
+
+### please
+
+That’s right. At Zendesk, you can skip the P-word. It takes up space in our UI
+and can come across as insincere.
+
+<BestPractice>
+  <Do>
+    <p>Your password was reset. Enter a new password.</p>
+  </Do>
+  <Dont>
+    <p>Your password was reset. Please enter a new password.</p>
+  </Dont>
+</BestPractice>
+
+### sorry
+
+We generally don’t apologize when things go wrong. When there’s an error, tell
+the user what happened and give them clear directions on what they can do next.
+
+Instead we save apologies for meaningful situations: when something goes really
+wrong or when Zendesk as a company has made a serious error, like deleting an
+account.
+
+### we
+
+Avoid we unless referring to a specific group of people. Don’t use we to refer
+to Zendesk the company or Zendesk the software. Similarly, avoid using our. See
+[Pronouns](/content/grammar#pronouns).
+
+<hr />
+
+## People
+
+The people in our Zendesk neighborhood. All of the people in the neighborhood
+have a job to do.
+
+### account owner
+
+An account owner is a type of administrator. The account name is associated with
+this person's name, usually the person who created the account. Just like The
+Highlander, there can be only one. For example, “Only the account owner has
+permission to make subscription changes.”
+
+### admin, administrator
+
+Admins have additional access to the Zendesk experience like setting up the
```

Comment:
_Admins_ have additional access to the Zendesk experience like setting up the workflows, adding users, and defining automations. We recommend _admins_ instead of _administrators_. It’s friendlier and shorter. For example, “Allow admins to
tag users.”